others/Hangman_game/Hangman.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <string.h>
+#include <fstream>
+#include <time.h>
+using namespace std;
+void print_string(char b[], int k, int len)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.